src/App.js

```diff
@@ -1,25 +1,96 @@
-import logo from './logo.svg';
 import './App.css';
+import React, { useState, useEffect } from 'react';
+export default App;
```

Comment:
this export should go last. it's a matter of convention more than anything, but you cannot export something that was not declared yet (like in this case)